### PR TITLE
Update set-up-content-understanding.md

### PR DESCRIPTION
**Pull request by msbemba**
Made a slight navigation change as the assign more button is not available any more

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4024

File: microsoft-365/contentunderstanding/set-up-content-understanding.md [hunk 1]
```diff
@@ -89,9 +89,9 @@ To assign licenses:
 
 2. Select the users that you want to license, and click **Manage product licenses**.
 
-3. Select **Assign more**.
+3. Click on **Apps** drop down.
```
Comment (JohanFreelancer9) [inline microsoft-365/contentunderstanding/set-up-content-understanding.md]:
```suggestion
3. Choose **Apps** from the drop-down menu.
```

File: microsoft-365/contentunderstanding/set-up-content-understanding.md [hunk 2]
```diff
@@ -89,9 +89,9 @@ To assign licenses:
...
 
-4. Select **SharePoint Syntex**. Under **Apps**, make sure **Common Data Service for SharePoint Syntex**, **SharePoint Syntex**, and **SharePoint Syntex - SPO type** are all selected.
+4. SelectSshow apps for  **SharePoint Syntex**. Under **Apps**, make sure **Common Data Service for SharePoint Syntex**, **SharePoint Syntex**, and **SharePoint Syntex - SPO type** are all selected.
```
Comment (JohanFreelancer9) [inline microsoft-365/contentunderstanding/set-up-content-understanding.md]:
```suggestion
4. Select **Show apps for  SharePoint Syntex**. Under **Apps**, make sure **Common Data Service for SharePoint Syntex**, **SharePoint Syntex**, and **SharePoint Syntex - SPO type** are all selected.
```

File: microsoft-365/contentunderstanding/set-up-content-understanding.md [hunk 3]
```diff
@@ -89,9 +89,9 @@ To assign licenses:
 
 2. Select the users that you want to license, and click **Manage product licenses**.
```
Comment (JohanFreelancer9) [inline microsoft-365/contentunderstanding/set-up-content-understanding.md]:
```suggestion
2. Select the users that you want to license, and choose **Manage product licenses**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - not sure, please confirm that my suggestions are technically correct.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks.

**Pull request merged**